blaze-core/src/test/scala/org/http4s/blaze/util/EntityBodyWriterSpec.scala

```diff
@@ -232,34 +228,30 @@ class EntityBodyWriterSpec extends Http4sSpec {
 
     "write a deflated resource" in {
       val p = resource through deflate()
-      p.runLog.map(_.toArray) must returnValue(DumpingWriter.dump(p))
+      p.runLog.map(_.toArray) must returnValue(DumpingWriter.dump(resource through deflate()))
```

Comment:
I'm confused.  You changed `dump(p)` to `dump(resource through deflate())` on master, and it passed?